packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
```

Comment:
Only able to login in Safari with email link (passkeys not working)

https://drive.google.com/file/d/1pbzvK7Rj5OglOMVDX1CLMB2o6nxG9G0W/view?usp=sharing